pkg/debuggers/controller.go

```diff
@@ -247,6 +280,34 @@ func (d *DebugController) tryToAttach(da *v1.DebugAttachment) error {
 	return nil
 }
 
+// uses the kubesquash debug approach
+func (d *DebugController) tryToAttachPod(da *v1.DebugAttachment) error {
+	ksConfig := kscmd.SquashConfig{
+		TimeoutSeconds: 300,
+		Machine:        true,
+		InCluster:      true,
+		DebugServer:    true,
+
+		DebugContainerVersion: version.ImageVersion,
+		DebugContainerRepo:    version.ImageRepo,
+
+		CRISock: "/var/run/dockershim.sock",
+
+		Debugger:  da.Debugger,
+		Namespace: da.Metadata.Namespace,
+		Pod:       da.Pod,
+		Container: da.Image,
+	}
+	debPod, err := kscmd.StartDebugContainer(ksConfig)
+	// TODO(mitchdraft) - refactor once old squash functionality is removed
+	if err != nil {
+		return err
+	}
+	debUrl := fmt.Sprintf("%v:%v", debPod.ObjectMeta.Name, 1235)
```

Comment:
this can be changed (should include the namespace) (extension hard coded to "squash" namespace right now but ought not be)